server/game/cards/characters/04/myrcellabaratheon.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('underscore');
+const DrawCard = require('../../../drawcard.js');
+
+class MyrcellaBaratheon extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => (
+                !this.getKingsInPlay() &&
+                this.game.currentChallenge &&
+                this.game.currentChallenge.challengeType === 'power'),
+            match: this,
+            effect: ability.effects.doesNotKneelAsDefender()
+        });
+
+        this.persistentEffect({
+            condition: () => !this.getKingsInPlay(),
+            match: this,
+            effect: ability.effects.addKeyword('Renown')
+        });
+    }
+
+    getKingsInPlay() {
```

Comment:
This function is incorrect. `_.each` will loop through the cards, but does not return the list of filtered cards. Additionally, you're not using the returned cards, only checking whether there are any. Finally, using `=> { ... }` opens up a lambda function which you need to explicitly return a value from, whereas `=> ( ... )` will implicitly return the result of whatever expression is in parentheses.

I would suggest rewriting this to be `areNoKingsInPlay` that returns a boolean instead using either `_.any` or `_.all`. e.g.
```javascript
areNoKingsInPlay() {
    return !_.any(this.game.getPlayers(), player => {
        return player.cardsInPlay.any(card => card.getType() === 'character' && card.hasTrait('King'));
    });
}
```

(probably want to double check the logic + syntax there, just quickly typing this here and not somewhere that would reveal obvious errors)